com.unity.uiwidgets/Runtime/painting/matrix_utils.cs

```diff
@@ -92,10 +144,28 @@ public static class MatrixUtils {
 
             return matrix;
         }
+        
+        public static Matrix3 toMatrix3(this Matrix4 matrix4) {
```

Comment:
why do we still need Matrix3 here? In my point of view, we can make all matrix3 to matrix4, and then convert matrix4 to uiMatrix?

Reply:
The engine part is still using matrix3